Bottom: 7c4de4aac1797ece32635cbdc40f156326b05a2c
Top:    1fff87d8d682bae7c7866c41e8b83f2a572394ad
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-16 16:11:18 +0100

Remove "=" shortcut to the djopt() thingy which keeps eating my design

ARR!! how much work have I lost to this key binding!










---

diff --git a/src/gpcb-menu.res b/src/gpcb-menu.res
index 34bc78c..99d41d5 100644
--- a/src/gpcb-menu.res
+++ b/src/gpcb-menu.res
@@ -326,13 +326,11 @@ MainMenu =
    {"Rip up all auto-routed tracks" RipUp(All)}
    -
    {"Optimize routed tracks"
-    {"Auto-Optimize" djopt(auto)  a={"Shift-=" "Shift<Key>="}}
     {"Debumpify" djopt(debumpify) }
     {"Unjaggy" djopt(unjaggy) }
     {"Vianudge" djopt(vianudge) }
     {"Viatrim" djopt(viatrim) }
     {"Ortho pull" djopt(orthopull) }
-    {"Simple optimization" djopt(simple)  a={"=" "<Key>="}}
     {"Miter" djopt(miter) }
     {"Puller" a={"Y" "<Key>y"} Puller() }
     {"Global Puller"
